dataprofiler/profilers/graph_profiler.py

```diff
@@ -57,6 +57,17 @@ def __init__(self, data, options=None):
             "categorical_distribution": GraphProfiler._update_categorical_distribution,
         }
 
+    def __add__(self, other):
+        """
+        Merge two Graph profiles together overriding the `+` operator.
+
+        :param other: graph profile being added to this one.
+        :type other: GraphProfiler
+        :return: merger of the two profiles
+        :rtype: GraphProfiler
+        """
+        raise NotImplementedError
```

Comment:
need to use it as a func: `NotImplementedError`  -> `NotImplementedError(description)`

Reply:
Also, will need a test to ensure the proper description

Reply:
+100

Reply:
Added